src/widget.rs

```diff
@@ -40,6 +44,13 @@ pub trait WidgetExtManual: 'static {
 
     fn connect_unmap_event<F: Fn(&Self, &Event) -> Inhibit + 'static>(&self, f: F) -> SignalHandlerId;
 
+    fn add_tick_callback<P: Fn(&Widget, &gdk::FrameClock) -> bool + 'static>(
```

Comment:
Should return a `glib::Continue` instead of a `bool`

Reply:
Hm, body better added to `impl<O: IsA<Widget>> WidgetExtManual for O {`
Also in current case IMHO first callback's parameter better be `&Self`.

Reply:
Sorry about "moving body" part, its already there, I mistakes it with multi-line definition

Reply:
Done both